api/routes/cartRouter.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+
+const { cartController } = require("../controllers");
+const { checkToken } = require("../middlewares/auth");
+
+const router = express.Router();
+
+router.post("/", checkToken, cartController.createCart);
```

Comment:
```suggestion
router.post("", checkToken, cartController.createCart);
```
REST API를 설계할 때에는, uniform interface 원칙에 따라 엔드포인트의 끝에는 (`/`) 쓰는 것을 지양하고 있습니다.